### DISCUSSION TITLE
disable 1 gpu from mining

### DISCUSSION THREAD

**Discussion by Rhacknar** [Lolliedieb/lolMiner-releases#560] [Q&A]
I mine often with 3 gpus i would like only 2 gpus to be mineing so i can use my main one for playing games while im mineing. i figured id find out if there was a way to disconnect one gpu from the miner rather than having to build a whole seperate pc for mineing

**Comment by Rhacknar** [comment]
Ok so i found my own solution i created a --help bat file and found all commands took me a minute to test and figure out how to do it. So at the end of script or anywhere as far as i know but after !WALLET! i put -- devices 0,1 this selected only those 2 gpus to run and disabled the third. for some reason my gpu that is gpu 1 on my radeon software ie main is actualy device 2 on my miner i guess it goes by hashrate maybe? either way i disabled my main made a new bat file and named it Mine_eth2gpus.bat so that i dont have to retype it out later so i have the option to switch between bats when im gaming and when im not.

**Reply by rkneisley** [reply]
Worked like a charm, I put it right after the wallet. Thank you
